opm/simulators/linalg/FlexibleSolver.hpp

```diff
@@ -33,6 +33,17 @@
 namespace Dune
 {
 
+template<class C>
+struct IsComm : std::false_type
+{};
+
+template<>
+struct IsComm<Dune::Amg::SequentialInformation> : std::true_type
+{};
+
+template<class Index>
+struct IsComm<Dune::OwnerOverlapCopyCommunication<Index>> : std::true_type
+{};
```

Comment:
I'm probably missing something simple, but this construction now generates a build failure for me in a sequential/debug/dune-2.6.0 configuration:
```
In file included from /home/bska/work/opm/src/opm/opm-simulators/tests/test_flexiblesolver.cpp:31:0:
/home/bska/work/opm/src/opm/opm-simulators/opm/simulators/linalg/FlexibleSolver.hpp:45:21: error: ‘OwnerOverlapCopyCommunication’ is not a member of ‘Dune’
 struct IsComm<Dune::OwnerOverlapCopyCommunication<Index>> : std::true_type
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/bska/work/opm/src/opm/opm-simulators/opm/simulators/linalg/FlexibleSolver.hpp:45:21: note: suggested alternative: ‘OwnerOverlapCopyAttributeSet’
 struct IsComm<Dune::OwnerOverlapCopyCommunication<Index>> : std::true_type
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                     OwnerOverlapCopyAttributeSet
/home/bska/work/opm/src/opm/opm-simulators/opm/simulators/linalg/FlexibleSolver.hpp:45:51: error: template argument 1 is invalid
 struct IsComm<Dune::OwnerOverlapCopyCommunication<Index>> : std::true_type
                                                   ^~~~~
/home/bska/work/opm/src/opm/opm-simulators/opm/simulators/linalg/FlexibleSolver.hpp:45:56: error: expected unqualified-id before ‘>’ token
 struct IsComm<Dune::OwnerOverlapCopyCommunication<Index>> : std::true_type
                                                        ^~
/home/bska/work/opm/src/opm/opm-simulators/opm/simulators/linalg/FlexibleSolver.hpp:46:3: warning: extra ‘;’ [-Wpedantic]
 {};
   ^
In file included from /home/bska/work/opm/inst/dune/2.6.0/dbg/seq/normal/include/dune/istl/paamg/construction.hh:11:0,
```
Please let me know if there's any additional information I can provide.

Reply:
Soory. See #2515 for a guessed.

Reply:
Undermines my impression that compilers a normally right (and not me).

Reply:
> See #2515 for a guessed fix.

Thanks.  PR #2525 does indeed restore my sequential build.  I appreciate the quick turn-around.